### PR TITLE
Fix 5XX/6XX fields and add generation of 901 field

### PR DESCRIPTION
**Pull request by aatuny**

### CODE REVIEW

File: frontend/js/config/merge-profiles/fennica.js [hunk 1]
```diff
@@ -92,6 +92,9 @@ import TargetRecord from '../target-record';
 import * as subrecordMergeTypes from '../subrecord-merge-types';
 import { hyphenate } from 'isbn-utils';
 
+var date = new Date().toISOString().replace(/-/g, '').replace(/T.*/, '');
```
Comment (natlibfi-arlehiko) [inline frontend/js/config/merge-profiles/fennica.js]: [Moment](http://momentjs.com/docs/) is a great module for datetime manipulation:
```js
const date = moment().format('YYYYMMDD');
```

File: frontend/js/config/merge-profiles/fennica.js [hunk 2]
```diff
@@ -92,6 +92,9 @@ import TargetRecord from '../target-record';
...
+var new901 = 'SU' + date;
```
Comment (natlibfi-arlehiko) [inline frontend/js/config/merge-profiles/fennica.js]: Use string templating: `const new901 = `SU${date}`;

File: frontend/js/config/merge-profiles/fennica.js (hunk 1 above)
Comment (natlibfi-arlehiko) [inline frontend/js/config/merge-profiles/fennica.js]: [Moment](http://momentjs.com/docs/) is a great module for datetime manipulation. The variable initialization can be reduced to a one-liner:
```js
const new901 = `SU${moment().format('YYYYMMDD')}`;
```

Comment (aatuny) [reply frontend/js/config/merge-profiles/fennica.js]: Fixed to use Moment.

File: frontend/js/config/merge-profiles/fennica.js [hunk 3]
```diff
@@ -92,6 +92,10 @@ import TargetRecord from '../target-record';
 import * as subrecordMergeTypes from '../subrecord-merge-types';
 import { hyphenate } from 'isbn-utils';
 
+var moment = require('moment');
```
Comment (natlibfi-arlehiko) [inline frontend/js/config/merge-profiles/fennica.js]: Import `moment` like this: `import moment from 'moment';`

Also, rebase the branch to merge commits to a single commit

Comment (aatuny) [reply frontend/js/config/merge-profiles/fennica.js]: Corrected `moment` to be imported correctly. Also commits should be rebased now to single commit.

**Pull request merged**